local-volume/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,50 @@
+common:
+#
+# Defines the namespace where provisioner runs
+#
+  namespace: default
+configmap:
+#
+# Defines the name of configmap used by Provisioner
+#
+  configMapName: "local-provisioner-config"
+#
+# Defines storage classes in a format:
+#
+#  storageClass:
+#    - {storage class name}:
+#        hostDir: "{path on the host where local volume of this storage class are mounted}"
+#        mountDir: "{Optional path where a local volume will be mounted inside of a container}"
+#
+# Note: At least one storage class must be configured
+  storageClass:
+    - fast-disks:
+        hostDir: /mnt/fast-disks
+        mountDir: /local-fast-disks
```

Comment:
Let's leave out the mountDir in the example since it's optional and I can't think of a common reason why a user would need to specify it.

Reply:
done